devtools-ci-index.yaml

```diff
@@ -898,6 +898,12 @@
             timeout: '30m'
             svc_name: keycloak-server
         - '{ci_project}-{git_repo}':
+            git_organization: almighty
+            git_repo: keycloak
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_run_tests.sh'
+            timeout: '20m'
```

Comment:
@alexeykazakov I know Alexey... but waiting 1h 20 minutes for some tests to finish. It is a ..., I will test some alternatives to see if we can speed it up.